tooling/bundler/src/bundle/common.rs

```diff
@@ -172,9 +172,12 @@ impl CommandExt for Command {
         if let Ok(0) = tauri_utils::io::read_line(&mut stdout, &mut buf) {
           break;
         }
-        debug!(action = "stdout"; "{}", String::from_utf8_lossy(&buf));
+        let mut line = String::from_utf8_lossy(&buf);
+        if line.ends_with('\n') {
+          line.to_mut().pop();
+        }
+        debug!(action = "stdout"; "{line}");
```

Comment:
How about just always ignoring the last character? since `read_line` will wait until `\n`
```suggestion
        debug!(action = "stdout"; "{}",  &line[0..buf.len()]);
```

Reply:
I think that is slower (isn't that creating another string?)

Reply:
Updated the snippet, it is just taking a reference

Reply:
I'd also suggest using `String::new` with a `BufReader` to read the lines into a string already, slice into it when printing and then extending the `lines` vector with the value of `string.as_bytes()`